Create 2020-08-26-first-post.md

intjnotinfp opened the pull request.

Commit:
Create 2020-08-26-first-post.md

eah13 commented on posts/intjnotinfp/2020-08-26-first-post.md:
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: <intjnotinfp>
```
The `<>` brackets aren't needed. No worries- using brackets in instructions  to indicate a 'variable' should go in there is an idiom you'll become more familiar with over time.

eah13 reviewed (changes requested):
I see two problems with the filename of your file: one is the underscore is missing in `_posts` and the second, which was an omission in the instructions, is we need your filename to have something unique in it in addition to your username.

This review says 'request changes' but everygthing's fine. We'll go over all of this in class.

Commit:
Update 2020-08-26-first-post.md

removed brackets, unable to change (directory? folder) though. needs further review.

Commit:
Rename posts/intjnotinfp/2020-08-26-first-post.md to _posts/intjnotinfp/2020-08-26-first-post.md

fixed _posts issue!

ghost commented:
@eah13 I changed the file name. Now ready for final review.

Commit:
Rename 2020-08-26-first-post.md to 2020-08-26-intjnotinfp-first-post.md

added unique title by including username

eah13 reviewed (approved):
Looks great now!
